frontend/src/store/actions/testScraperFormActions.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+export const INITIALIZE_URL_SUBMIT = 'INITIALIZE_URL_SUBMIT';
+export const COMPLETE_URL_SUBMIT = 'COMPLETE_URL_SUBMIT';
+export const SUBMIT_URL_ERROR = 'SUBMIT_URL_ERROR';
+
+export const sendURL = newURL => {
+	return dispatch => {
+		//Again, action to indicate an API call is about to be made, this time for a POST
+		dispatch({ type: INITIALIZE_URL_SUBMIT });
+		//Below, you're making the POST call to the API, with newURL as the object youre sending.
+		axios
+			.post('http://127.0.0.1:8000/api/users/pages/', newURL)
```

Comment:
Sorry gave wrong endpoint for sending URL to correct one will be `http://127.0.0.1:8000/api/scrape/`